src/test/java/zipCodes/GetUserTest.java

```diff
@@ -0,0 +1,52 @@
+package zipCodes;
+
+import by.issoft.ResponseEntity;
+import by.issoft.client.UserClient;
+import by.issoft.dto.Sex;
+import by.issoft.dto.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+public class GetUserTest {
+
+    private static final int RESPONSE_CODE = 200;
+    private UserClient userClient;
+
+    @BeforeEach
+    void initUserClient() {
+        userClient = new UserClient();
+    }
+
+    @Test
+    void getUserTest() {
+        ResponseEntity<List<User>> users = userClient.getUsers();
+        Assertions.assertEquals(RESPONSE_CODE, users.getStatusCode());
+        Assertions.assertNotNull(users.getBody());
+    }
+
+    @Test
+    void getOlderUsersTest() {
+        Integer ageToCheck = 40;
```

Comment:
try to use primitives wherever possible. String.valueOf(int) in 33 row
fyi
https://stackoverflow.com/questions/5199359/why-do-people-still-use-primitive-types-in-java